tsconfig.json

```diff
@@ -0,0 +1,100 @@
+{
+  "compilerOptions": {
+    /* Visit https://aka.ms/tsconfig.json to read more about this file */
+
+    /* Projects */
+    // "incremental": true,                              /* Enable incremental compilation */

```

Comment:
Not blocking, but IMO TypeScript ships with autocomplete/documentation on these so I find the commented values to be noisy. But don’t care strongly enough to request they be removed; I’ll let the author decide.